graylog2-server/src/main/java/org/graylog2/migrations/V20161216123500_DefaultIndexSetMigration.java

```diff
@@ -73,7 +73,7 @@ public void upgrade() {
             final String analyzer = elasticsearchConfiguration.getAnalyzer();
             final IndexSetConfig updatedConfig = indexSetConfig.toBuilder()
                     .indexAnalyzer(analyzer)
-                    .indexTemplateName(elasticsearchConfiguration.getTemplateName())
+                    .indexTemplateName(indexSetConfig.indexPrefix() + "-template")
```

Comment:
As written earlier, the default index set needs the template name from the old ES configuration options. It might have been modified by the user and our default template for new indices would break that.